cmd/kubeadm/app/cmd/upgrade/diff_test.go

```diff
@@ -124,3 +124,67 @@ func TestRunDiff(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateManifests(t *testing.T) {
+	// Create valid manifest paths
+	apiServerManifest, err := createTestRunDiffFile([]byte{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.Remove(apiServerManifest)
+	controllerManagerManifest, err := createTestRunDiffFile([]byte{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.Remove(controllerManagerManifest)
+	schedulerManifest, err := createTestRunDiffFile([]byte{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.Remove(schedulerManifest)
+	// Create a directory path that does not exist
+	notExistDirPath := "./foobar123456"
+	// Create a file path that does not exist
+	notExistFilePath := notExistDirPath + "/xxxxx.yaml"
+
+	testCases := []struct {
+		name          string
+		args          []string
+		expectedError bool
+	}{
+		{
+			name:          "valid: valid manifest path",
+			args:          []string{apiServerManifest, controllerManagerManifest, schedulerManifest},
+			expectedError: false,
+		},
+		{
+			name:          "invalid: one is empty path",
+			args:          []string{apiServerManifest, controllerManagerManifest, ""},
+			expectedError: true,
+		},
+		{
+			name:          "invalid: manifest path is directory",
+			args:          []string{"/"},
```

Comment:
```suggestion
			args:          []string{"./"},
```

unit tests are sometimes run by a root user.
let's not do anything with the host root `/`, so that we don't write or delete files by mistake there.